sdk/src/client/api/block_builder/mod.rs

```diff
@@ -43,12 +44,14 @@ impl ClientInner {
                 issuance.latest_finalized_slot,
                 issuer_id,
             ),
-            // TODO: burned mana calculation
-            BlockBody::build_basic(issuance.strong_parents()?, 0)
+            BlockBody::build_basic(issuance.strong_parents()?)
                 .with_weak_parents(issuance.weak_parents()?)
                 .with_shallow_like_parents(issuance.shallow_like_parents()?)
                 .with_payload(payload)
-                .finish_block_body()?,
+                .finish_block_body_with_minimum_mana_amount(
```

Comment:
I would still do as Thoralf suggested as this doesn't allow passing a specific value

Reply:
there is `with_max_burned_mana`

Reply:
Specific value could be provided with `with_max_burned_mana(mana)` and then `.finish_block_body()`
Not sure what I like more

Reply:
Probably easier to forget setting this value to anything with the functions like in this PR now

Reply:
Ah yeah I forgot `with_max_burned_mana`, but I agree, super easy to not set it and then it defaults to 0 which is bad. I would actually remove `with_max_burned_mana`.

Reply:
I mean, maybe not remove it as we still have `with_amount` for outputs but you get me

Reply:
yeah, I wouldn't remove it